scene/gui/line_edit.cpp

```diff
@@ -613,9 +613,7 @@ void LineEdit::_notification(int p_what) {
 #endif
 		case NOTIFICATION_RESIZED: {
 
-			if (expand_to_text_length) {
-				window_pos = 0; //force scroll back since it's expanding to text length
-			}
+			window_pos = 0; //force scroll back since it's expanding to text length
```

Comment:
The comment is no longer accurate.

Reply:
Yes you are right. Could you please remove it from master since it is already merged there?

@akien-mga , @YeldhamDev Thank you for reviewing my pull request.

Reply:
I've removed it in this pull: #28913.

And you're very welcome!